TrackletAlgorithm/MatchEngineTop.h

```diff
@@ -0,0 +1,11 @@
+#ifndef TrackletAlgorithm_MatchEngineTop_h
+#define TrackletAlgorithm_MatchEngineTop_h
+
+#include "MatchEngine.h"
+
+void MatchEngineTop(const BXType bx, BXType& bx_o,
+					const VMStubMEMemory<MODULETYPE, NBITBIN>& inputStubData,
+					const VMProjectionMemory<PROJECTIONTYPE>& inputProjectionData,
+					CandidateMatchMemory& outputCandidateMatch);
+
+#endif
```

Comment:
@aperloff Please add a newline at the end of the file.

Reply:
Done